packages/dds/map/src/map.ts

```diff
@@ -115,6 +115,50 @@ export class SharedMap extends SharedObject<ISharedMapEvents> implements IShared
 		return new MapFactory();
 	}
 
+	/**
+	 * Test API with comprehensive TSDoc documentation.
+	 *
+	 * @remarks
+	 * This API is designed to showcase all possible features of TSDoc and API Extractor.
+	 *
+	 * @defaultValue 'N/A'
+	 * @decorator TEST
```

Comment:
We don't currently support docs gen for decorators. This is something we would need to add to our api-documenter library. I believe we intend to start using decorators after our forthecoming typescript version update, so this would be good to add. If we need this soon, we should sync up  - should be easy to add support!